Doc/library/pathlib.rst

```diff
@@ -1539,50 +1539,33 @@ Creating files and directories
 Copying, renaming and deleting
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. method:: Path.copy(target, *, follow_symlinks=True, preserve_metadata=False)
+.. method:: Path.copy(target, *, follow_symlinks=True, \
+                      preserve_metadata=False, dirs_exist_ok=False, \
+                      ignore=None, on_error=None)
 
-   Copy the contents of this file to the *target* file. If *target* specifies
-   a file that already exists, it will be replaced.
+   Copy this file or directory tree to the given *target*. If both paths are
+   existing files, the target file will be replaced.
```

Comment:
The path we're copying should always exist, maybe something like "When copying a file, *target* will be replaced if it is an existing file."?